README.md

```diff
@@ -24,7 +24,20 @@ Gogs - Go Git Service [![Build Status](https://travis-ci.org/gogits/gogs.svg?bra
 
 ## Purpose
 
-The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. With Go, this can be done with an independent binary distribution across **ALL platforms** that Go supports, including Linux, Mac OS X, Windows and ARM.
+The goal of this project is to make the easiest, fastest, and most painless way of setting up a self-hosted Git service. With Go, this can be done with an independent binary distribution across [**ALL platforms** that Go supports](https://github.com/golang/go/wiki/MinimumRequirements) ([Go Build Dashboard](https://build.golang.org/)), including operating systems like:
```

Comment:
`The goal of this project is to make the easiest, fastest and most painless way of setting up a self-hosted web Git service. With Golang, this can be done with an independent binary distribution across [**ALL platforms** that Go supports](https://github.com/golang/go/wiki/MinimumRequirements) ([Go Build Dashboard](https://build.golang.org/)), including operating systems like: